src/domain/synthetics/fees/utils/priceImpact.ts

```diff
@@ -15,11 +15,13 @@ export function getPriceImpactByAcceptablePrice(p: {
   const shouldFlipPriceDiff = isIncrease ? !isLong : isLong;
 
   const priceDelta = markPrice.sub(acceptablePrice).mul(shouldFlipPriceDiff ? -1 : 1);
-  const acceptablePriceDeltaBps = getBasisPoints(priceDelta, p.indexPrice);
+  const acceptablePriceDeltaBps = markPrice.isZero() ? BigNumber.from(0) : getBasisPoints(priceDelta, markPrice);
```

Comment:
let's use BN_ZERO here, perhaps it might optimize some selector one day

Reply:
✅